packages/jest-expo/jest-preset.js

```diff
@@ -27,15 +27,55 @@ jestPreset.transform['\\.[jt]sx?$'] = [
   { caller: { name: 'metro', bundler: 'metro', platform: 'ios' } },
 ];
 
-const defaultAssetNamePattern = '^.+\\.(bmp|gif|jpg|jpeg|mp4|png|psd|svg|webp)$';
-if (!jestPreset.transform[defaultAssetNamePattern]) {
-  console.warn(`Expected react-native/jest-preset to define transform[${defaultAssetNamePattern}]`);
-} else {
-  delete jestPreset.transform[defaultAssetNamePattern];
-}
```

Comment:
@gabrieldonadel and I couldn't figure out what this was for. as far as we could tell, this code is redundant because we replace all of the transforms with our own transforms immediately after this. I'm not sure what the value is here of warning - perhaps to give us a heads up that something changed in react-native? anyways it dates back to sdk 33 so probably nobody knows.